src/com/fundation/search/model/FileClass.java

```diff
@@ -0,0 +1,63 @@
+package com.fundation.search.model;
+
```

Comment:
line 1, must be after copyright, the same for your other classes.

Reply:
Done